index.html

```diff
@@ -1,12 +1,256 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Conference Page</title>
-</head>
-<body>
-    <h1>Conference Page</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="/images/icons/favicon.ico" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/earlyaccess/nanumgothic.css"
+      type="text/css"
+      media="all"
+    />
+    <title>CC Global Summit 2015</title>
+  </head>
+
+  <body>
+    <nav class="top-menu">
+      <ul class="top-menu-item">
+        <li>
+          <a href="#">
+            <img src="images/icons/facebook.png" alt="Facebook"
+          /></a>
+        </li>
+        <li>
+          <a href="#"> <img src="images/icons/twitter.png" alt="Twitter" /></a>
+        </li>
+        <li>
+          <a href="#">English</a>
+        </li>
+        <li>
+          <a href="#">My Page</a>
+        </li>
+        <li>
+          <a href="#">Logout</a>
+        </li>
+      </ul>
+    </nav>
+
+    <div id="toggle-menu" class="hidden">
+      <div class="close-wrapper clearfix" id="close-menu">
+        <img src="images/icons/close.png" alt="Close" />
+      </div>
+      <nav class="toggle-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="https://amrendrakind.github.io/conferencePage_capstoneProject/" class="nav-link active">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Join</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">News</a>
+          </li>
```

Comment:
- [ ] Please, consider fixing the mobile menu on both pages. As you can see on the gif below, it is not fixed and does not cover the whole page and there is a space on the bottom, where we can see a scrolling page.
![2022-03-18 22 46 53](https://user-images.githubusercontent.com/49064106/159069690-86a69e48-5d02-4d28-9c04-92ea86c93a65.gif)